src/modules/toolbox/components/drawToolContent/DrawToolContent.js

```diff
@@ -7,7 +7,7 @@ import css from './drawToolContent.css';
 import { StyleSizeTypes } from '../../../../services/domain/styles';
 import { clearDescription, clearText, finish, remove, reset, setDescription, setStyle, setType } from '../../../../store/draw/draw.action';
 import { unsafeHTML } from 'lit-html/directives/unsafe-html.js';
-import { AssetSourceType, getAssetSource, hexToRgb } from '../../../map/components/olMap/olStyleUtils';
+import { AssetSourceType, getAssetSource, hexToRgb } from '../../../olMap/utils/olStyleUtils';
```

Comment:
`hexToRgb`, `rgbToHex` etc. are better kept in a separate ./src/utils/colors.js, aren't they?
`AssetSourceType`, `getAssetSource` in  the same way in ./src/utils/asset.js. 

There is no obvious direct relation to the olMap-module, so they can be extracted.

Reply:
Absolutely correct. It violates our encapsulated module rule (see Readme). But let's fix that by a separate PR. Would you open an issue for that?

Reply:
Sure